### DISCUSSION TITLE
Same Hash Value for Different Images

### DISCUSSION THREAD

**Discussion by Cyanic** [pwlmc/imghash#148] [Q&A] [answered]
I'm hashing 2 png's that are different yet I'm getting the same hash value is this possible? Or am I doing something wrong?

**Comment by Cyanic** [comment, marked as answer]
Looks like increasing the bits to 16 might have solved this.

**Reply by pwlmc** [reply]
@Cyanic yes, it's totally possible that you will get [the same hash for different images](https://en.wikipedia.org/wiki/Hash_collision). The probability of it increases the shorter the hash is so increasing the hash length will fix the issue.
